### DISCUSSION TITLE
Best settings for Arma 3 and 1 Problem

### DISCUSSION THREAD

**Discussion by YilizGorilla** [Blinue/Magpie#710] [Q&A]
What are the best configurations for Arma 3 to get more fps. 

And when I start the upscaling there appear 2 problems. First of all my Arma 3 expands to my second monitor and second the in-game Overlay tells me I get likely 200FPS but the Game says only 30-50FPs

**Comment by Devilmonk7402** [comment]
Hey, 
So I'm also quite new to the program as well, so definitely do wait until the author, or more experienced / qualified users reply. (I only a few days ago; mostly just reading through all the wiki, discussion, and issue threads, etc). 

With that being said, regarding the second issue about your FPS counter, I believe that issue is due to:


> **What is the frame rate shown in Magpie?**
> Magpie displays its own frame rate instead of the game's. Based on the non-intrusive principle, Magpie cannot know the game's frame rate. It is recommended to use tools such as RTSS to display the game frame rate, and their overlays can generally be captured by Magpie.

> **Duplicate-monitoring**
> When displaying performance monitor like RTSS (Rivatuner Statistics Server), there might be 2 OSD layers displayed with Magpie scaling. This is caused by d3d, the screen capture method since v0.7.0. It will be captured by RTSS as well. You can fix this issue by adding it to the blacklist. 

^ From the [wiki > FaQ](https://github.com/Blinue/Magpie/wiki/FAQ%20(EN))

If you're still having that first issue with the game spanning two screens, I'm not quite sure. I have a _feeling_ it's due to how the scaling factor is setup, since I believe by default it tries to match it to your display resolution; but perhaps that meant the _entire resolution_ across both monitors(?) 

**But that's only my guess anyway.** I'm not familiar with how their scaling factor options works. Honestly I've been trying to look into that for a while now, since their ReadMe says it's better to have it setup to your own match your own setup/resolution; but alas, I'm still at a loss for how it works, or what raising/lowing the numbers do. But if you want to play around with it, I'll post some pictures of how you can get to that setting:

![PXL_20230831_065951164.jpg](https://github.com/Blinue/Magpie/assets/95123219/b09655f9-0707-4570-a16b-013495833253)

![PXL_20230831_070011107.jpg](https://github.com/Blinue/Magpie/assets/95123219/793ae392-3d34-449a-9e9f-787415aa0491)

![PXL_20230831_070034838.jpg](https://github.com/Blinue/Magpie/assets/95123219/bd8b5092-abe1-48f3-8063-adf1246dae56)

![PXL_20230831_070051917.jpg](https://github.com/Blinue/Magpie/assets/95123219/0e0ce04e-716d-4527-be84-b9e2102bf952)
